Library/Formula/libsamplerate.rb

```diff
@@ -26,6 +26,19 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
+    rm_f "#{bin}/sndfile-resample" if build.without? "libsndfile"
+  end
+
+  def caveats
+    s = ""
```

Comment:
To be fair we use `s = ""` in every other formula that sets a blank initial caveat AFAIK.